node/mdSchema.ts

```diff
@@ -189,6 +206,14 @@ export const schemas = [
           type: 'array',
           title: 'Default Price Tables',
         },
+        organizationCustomFields: {
+          type: 'array',
+          title: 'Organization Custom Fields',
+        },
+        costCenterCustomFields: {
+          type: 'array',
+          title: 'Cost Center Custom Fields',
+        },
```

Comment:
The setting schema isn't used for anything with Vbase, this entire section is going to be removed.

Reply:
Should I remove it? Or keep it for now?

Reply:
I'd remove just the lines you added, that should ensure it doesn't conflict when Paulo removes the section in his PR.

Reply:
✅Removed all lines dealing with custom fields